app/src/main/java/de/westnordost/streetcomplete/quests/bike_parking_cover/AddBikeParkingCover.java

```diff
@@ -0,0 +1,53 @@
+package de.westnordost.streetcomplete.quests.bike_parking_cover;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddBikeParkingCover extends SimpleOverpassQuestType
+{
+	@Inject public AddBikeParkingCover(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways, relations with amenity=bicycle_parking and access!=private and !covered";
+	}
+
+	@Override
+	public int importance()
+	{
+		return QuestImportance.MINOR;
+	}
+
+	public AbstractQuestAnswerFragment createForm()
+	{
+		AbstractQuestAnswerFragment form = new YesNoQuestAnswerFragment();
+		form.setTitle(R.string.quest_bicycleParkingCoveredStatus_title);
+		return form;
+	}
+
+	public void applyAnswerTo(Bundle answer, StringMapChangesBuilder changes)
+	{
+		String yesno = answer.getBoolean(YesNoQuestAnswerFragment.ANSWER) ? "yes" : "no";
+		changes.add("access", yesno);
+	}
+
+	@Override public String getCommitMessage()
+	{
+		return "Add whatever bicycle parking is protected from rain";
```

Comment:
I do not understand this wording. I'd say "Add bicycle parkings cover". The commit message is intended for people who look at the changesets and this will be people who know the OSM tags. (So they also know cover and know what is meant).